src/main/java/io/jenkins/plugins/orka/client/OrkaVMConfig.java

```diff
@@ -13,10 +13,19 @@ public class OrkaVMConfig {
     @SerializedName("orka_base_image")
     private String baseImage;
 
+    private int memory;
```

Comment:
I believe this should be string. The server can return `automatic`

Reply:
Btw lets unify automatic and auto across our of Orka's code.
Lets pick one of them and use it everywhere